packages/playwright/src/runner/loadUtils.ts

```diff
@@ -136,10 +136,10 @@ export async function createRootSuite(testRun: TestRun, errors: TestError[], sho
 
     // Filter file suites for all projects.
     for (const [project, fileSuites] of testRun.projectSuites) {
+      projectSuites.set(project, createProjectSuite(project, fileSuites));
```

Comment:
Can we avoid calling `createProjectSuite` twice? It does a deep clone, so I'd like to not call it more than necessary. We can probably move the `additionalFileMatcher` to be an option of `filterProjectSuite` to achieve that.

Reply:
sure thing! done uin https://github.com/microsoft/playwright/pull/32094/commits/1899d2745a6d856da518634aa1e8fe0a08cc5344